src/Codec/Xlsx/Types/SheetState.hs

```diff
@@ -0,0 +1,45 @@
+{-# LANGUAGE DeriveGeneric     #-}
+{-# LANGUAGE OverloadedStrings #-}
+
+module Codec.Xlsx.Types.SheetState where
```

Comment:
why does it deserve a separate module? It's rather short and doesn't contain much extra....

Reply:
Seemed tidier ; do you propose moving it to `Codec.Xlsx.Types.Common` ?

Reply:
It's related to `Worksheet` and that one lives in `Codec.Xlsx.Types`, that one is a bit large but that's the best I could propose...